src/classes/UTIL_Describe.cls

```diff
@@ -618,8 +618,7 @@ public class UTIL_Describe {
 
             if(getRecTypesMapByDevName('Account').values().size() > 1) {
                 return getRecTypesMapByDevName('Account').values()[1];
-            }
-            else {
+            }else {
```

Comment:
@skosgi1 SUPER tiny fix. put a space before the else here. } else {

Reply:
@lifewithryan My bad. fixed it.